LondonBritishLibrary/orient/BLorient795.xml

```diff
@@ -32,8 +35,804 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                     <adminInfo>
+                         <recordHist>
+                             <source>
+                                 <listBibl type="catalogue">
+                                     <bibl>
+                                         <ptr target="bm:Wright1877BM"/>
+                                         <citedRange unit="page">275a-277a</citedRange>
+                                     </bibl>
+                                 </listBibl>
+                             </source>
+                         </recordHist>
+                     </adminInfo>
+                 </additional>
+               <msPart xml:id="p1">
+                 <msIdentifier></msIdentifier>
+                 <msContents>
+                   <msItem xml:id="p1_i1">
+                     <locus from="1r"></locus>
+                     <title type="complete" ref="LIT1295Dersan"></title>
+                     <textLang mainLang="gez"></textLang>
+                     <note>The rubrics are omitted.</note>
+                   </msItem>
+                   </msContents>
+                   <physDesc>
+                       <objectDesc form="Leaf">
+                           <supportDesc>
+                               <support>
+                                   <material key="parchment"/>
+                               </support>
+                               <extent>
+                                   <note>Of a smaller size than <ref target="#p2"></ref>.</note>
+                               </extent>
+                           </supportDesc>
+
+                           <layoutDesc>
+                               <layout columns="2" writtenLines="17">
+                               </layout>
+                           </layoutDesc>
+                       </objectDesc>
+                       <handDesc>
+                           <handNote script="Ethiopic" xml:id="h1">
+                               <seg type="rubrication">Rubrics are omitted.</seg>
+                           </handNote>
+                       </handDesc>
+                   </physDesc>
+                   <history>
+                       <origin>
+                           <origDate>Of later date than <ref target="#p2"></ref>.</origDate>
+                       </origin>
+                   </history>
+                </msPart>
+                <msPart xml:id="p2">
+                  <msIdentifier></msIdentifier>
+               <msContents>
+                 <summary></summary>
+                 <msItem xml:id="p2_i1">
+                   <locus from="2r"></locus>
+                   <title type="complete" ref="LIT2461Testam" xml:lang="en">Testament
+                   of our Lord.</title>
+                   <textLang mainLang="gez"></textLang>
+
+                   <msItem xml:id="p2_i1.1">
+                     <locus from="2r"></locus>
+                     <title type="complete" xml:lang="en">Of the Holy Spirit.</title>
```

Comment:
Wright's chapters seem to correspond to the ones given in the work record. So you could maybe add the reference to all chapters in the title? And conversely, I would add the chapter titles in the work records, but that might be personal preference.